introduction/deprecated_features/README.md

```diff
@@ -6,15 +6,17 @@ When a feature is first deprecated, it may not yet be announced in which CSM ver
 been made, that information will be available on this page. For any deprecated features listed on this page that do not yet have an announced CSM
 version for their planned removal, customers are still strongly encouraged to make plans to migrate away from the deprecated feature.
 
-* [Removals](#removals)
-  * [Removals in CSM 1.4](#removals-in-csm-14)
-  * [Removals in CSM 1.5](#removals-in-csm-15)
-  * [Removals in CSM 1.6](#removals-in-csm-16)
-* [Deprecations](#deprecations)
-  * [Deprecated in CSM 1.3](#deprecated-in-csm-13)
-  * [Deprecated in CSM 1.2](#deprecated-in-csm-12)
-  * [Deprecated in CSM 1.0](#deprecated-in-csm-10)
-  * [Deprecated in CSM 0.9.3](#deprecated-in-csm-093)
+- [Deprecated Features](#deprecated-features)
+    - [Removals](#removals)
+        - [Removals in CSM 1.4](#removals-in-csm-14)
+        - [Removals in CSM 1.5](#removals-in-csm-15)
+        - [Removals in CSM 1.6](#removals-in-csm-16)
+    - [Deprecations](#deprecations)
+        - [Deprecated in CSM 1.5](#deprecated-in-csm-15)
+        - [Deprecated in CSM 1.3](#deprecated-in-csm-13)
+        - [Deprecated in CSM 1.2](#deprecated-in-csm-12)
+        - [Deprecated in CSM 1.0](#deprecated-in-csm-10)
+        - [Deprecated in CSM 0.9.3](#deprecated-in-csm-093)
```

Comment:
No need to include the page itself in the table of contents.

```suggestion
- [Removals](#removals)
    - [Removals in CSM 1.4](#removals-in-csm-14)
    - [Removals in CSM 1.5](#removals-in-csm-15)
    - [Removals in CSM 1.6](#removals-in-csm-16)
- [Deprecations](#deprecations)
    - [Deprecated in CSM 1.5](#deprecated-in-csm-15)
    - [Deprecated in CSM 1.3](#deprecated-in-csm-13)
    - [Deprecated in CSM 1.2](#deprecated-in-csm-12)
    - [Deprecated in CSM 1.0](#deprecated-in-csm-10)
    - [Deprecated in CSM 0.9.3](#deprecated-in-csm-093)
```

Reply:
This can be handled with a different ticket.

Reply:
Fixed.